contracts/secret-storage/SecretManager.sol

```diff
@@ -601,70 +625,26 @@ contract SecretManager is
         if(block.timestamp > userStorage[_secretId].endTimestamp)
             revert SecretManagerAlreadyTerminated();
 
-        _clearSecretData(_secretId);
+        userStorage[_secretId].endTimestamp = block.timestamp;
 
         emit SecretTerminated(_secretId);
```

Comment:
can you add timestamp to this event? Required for enclaves to compute billing details.